merchant-sdk/example-app/src/main/java/net/gini/android/merchant/sdk/exampleapp/review/ReviewActivity.kt

```diff
@@ -163,6 +162,8 @@ class ReviewActivity : AppCompatActivity() {
                 return@launch
             }
 
+            viewModel.giniMerchant.setDocumentForReview(documentId)
```

Comment:
I think this can be removed when merging with EC-60 - since the instantiation of the container fragment contains the documentId, setting the document for review can be done internally.

Reply:
I agree 👍